src/SolutionTemplate/UnoSolutionTemplate/iOS/UnoQuickStart.iOS.csproj

```diff
@@ -117,8 +117,8 @@
 	<ItemGroup>
 		<PackageReference Include="Uno.UI" Version="2.2.0" />
 		<PackageReference Include="Uno.UI.RemoteControl" Version="2.2.0" Condition="'$(Configuration)'=='Debug'" />
-		<PackageReference Include="Microsoft.Extensions.Logging.Console" Version="1.1.1" />
-		<PackageReference Include="Microsoft.Extensions.Logging.Filter" Version="1.1.1" />
+		<PackageReference Include="Microsoft.Extensions.Logging" Version="5.0.0" />
+    <PackageReference Include="Uno.Extensions.Logging.OSLog " Version="1.0.0-dev.3" />
```

Comment:
Why is the difference on iOS 😮 ?

Reply:
iOS logging though the console does not go to the system log anymore, which is generally a problem